src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -150,6 +150,21 @@ public static void TestForErrorInGraphQLResponse(string response, string message
             }
             else
             {
+                // Json Property in error which the holds the actual exception properties.
+                string PARENT_PROPERTY_ERROR = "error";
+
+                //Generate expected error object
+                JsonElement expectedErrorObj = JsonDocument.Parse(expected).RootElement.GetProperty(PARENT_PROPERTY_ERROR);
+                string expectedStatusCode = expectedErrorObj.GetProperty(PROPERTY_STATUS).ToString();
+                string expectedSubStatusCode = expectedErrorObj.GetProperty(PROPERTY_CODE).ToString();
+
+                if ("403".Equals(expectedStatusCode))
+                {
+                    // In case of forbidden requests (403 status code), responseBody is read as empty.
+                    Assert.IsTrue(string.IsNullOrEmpty(responseBody));
```

Comment:
Would this be true?  do we not return the error code/message in the body as well? maybe I'm mistaken and 403 Forbidden responses only return status code.

Reply:
No, we do return everything including message,statuscode,substatuscode for forbidden responses as well. But for 403 responses, this method here fails to parse the response https://github.com/Azure/data-api-builder/blob/9d413132936d8a605ca14765b5262b089f6b4e94/src/Service.Tests/SqlTests/SqlTestHelper.cs#L124

Reply:
However in postman or any other client, we only see that the request is forbidden.

Reply:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/34566234/227106262-da90babd-c9f1-4b4a-9429-e234c8da1493.png">

Reply:
> . But for 403 responses, this method here fails to parse the response

is that something that needs to be fixed?

Reply:
I am beginning to think that in case of 403 everything else is ignored by the server, (we don't see anything other than the status code in postman as well).

Reply:
No more a consideration since we are throwing the entire exception for 403 case as well.